db/models.py

```diff
@@ -1 +1,22 @@
 from django.db import models
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description= models.TextField(blank=True)
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Rase, on_delete=models.CASCADE)
+
+class  Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(unique=True, null=True)
```

Comment:
The `description` field in the `Guild` model is marked as `unique=True`. Typically, descriptions are not unique, so you might want to remove this constraint unless it's intentional.